backend/migrations/versions/999999999999_manually_add_delegation_for_one_of_users.py

```diff
@@ -0,0 +1,41 @@
+"""Manually add delegation for one of the users
+
+Revision ID: 999999999999
+Revises: 19f6b4f5ce2e
+Create Date: 2024-07-17 16:48:12.133033
+
+"""
+from alembic import op
+# from app.modules.common.delegation import get_hashed_addresses
+
+revision = "999999999999"
+down_revision = "19f6b4f5ce2e"
+branch_labels = None
+depends_on = None
+
+# primary = "0x2222222222222222222222222222222222222222" # redacted
+# secondary = "0x1111111111111111111111111111111111111111" # redacted
+
+# (hash1, hash2, hash3) = get_hashed_addresses(primary, secondary)
+# values of hashes below were obtained using code above; with appropriately set DELEGATION_SALT and DELEGATION_SALT_PRIMARY envs
+hash1 = "a4ed43527ceb61080a61e8661aa63fa7a266a313c08473db31fa1acafa32bd16"
+hash2 = "772d00c482a244e447c8d460661b552fb9b06d58e7903f3bbfa4e0e14dd92b99"
+hash3 = "fb610f25ff00d710de6578a2b684fdd2dcb353ed341e9f22a100af754ef36ae6"
+
+
+def upgrade():
+    query = f"INSERT INTO score_delegation (hashed_addr) VALUES ('{hash1}');"
+    print(query, flush=True)
+    op.execute(query)
+    query = f"INSERT INTO score_delegation (hashed_addr) VALUES ('{hash2}');"
+    print(query, flush=True)
+    op.execute(query)
+    query = f"INSERT INTO score_delegation (hashed_addr) VALUES ('{hash3}');"
```

Comment:
shouldn't we also save the antysibil score in the database after such delegations?

Reply:
fixed